mne_bids/write.py

```diff
@@ -1244,11 +1244,23 @@ def make_dataset_description(*, path, name, hed_version=None,
 
 
 @verbose
-def write_raw_bids(raw, bids_path, events_data=None, event_id=None,
-                   anonymize=None, format='auto', symlink=False,
-                   empty_room=None, allow_preload=False,
-                   montage=None, acpc_aligned=False,
-                   overwrite=False, verbose=None):
+def write_raw_bids(
+    raw,
+    bids_path,
+    *,
+    events=None,
+    events_data=None,
```

Comment:
For now we accept both parameters. Passing events_data will emit a FutureWarning.